src/VisualStudio/Core/Def/Implementation/FindReferences/TableEntriesSnapshot.cs

```diff
@@ -9,6 +9,9 @@ namespace Microsoft.VisualStudio.LanguageServices.FindUsages
 {
     internal partial class StreamingFindUsagesPresenter
     {
+        // Name of the key used to retireve the whole entry object.
+        internal const string SelfKeyName = "self";
```

Comment:
Is this a constant being shared with something else?

Reply:
It is just introduced and shared between FindUsagesTableControlEventProcessorProvider and StreamingFindUsagesPresenter

---
In reply to: [335225827](https://github.com/dotnet/roslyn/pull/39055#discussion_r335225827) [](ancestors = 335225827)